testapi.pm

```diff
@@ -1937,7 +1937,7 @@ sub data_url($) {
         return autoinst_url("/assets/repo/" . get_var($name));
     }
     if ($name =~ /^ASSET_\d$/) {
-        return autoinst_url("/assets/other/" . get_var($name));
+        return autoinst_url("/assets/other/" . (split /\//, get_var($name))[-1]);
```

Comment:
I was not able to quickly find what has caused this behavior, but would it simply work if we call
`autoinst_url(get_var($name))` as variable contains full path?

Reply:
I guess no, at least when in comes to WSL the appx file is stored in the pool.
`ASSET_1 | "/var/lib/openqa/pool/9/openSUSE-Tumbleweed-x64-Snapshot20200319.5.26.appx"`

Reply:
I am not sure

Reply:
One minor thing, as we already have `File::Basename` as dependency, I would propose to use it instead of split + slice.

Reply:
We also might want to update/add new test for `data_url`